go/bundle/bundle.go

```diff
@@ -10,24 +10,120 @@ import (
 	"net/http"
 	"net/url"
 	"regexp"
-	"strconv"
 	"strings"
 
-	"github.com/WICG/webpackage/go/signedexchange"
 	"github.com/WICG/webpackage/go/signedexchange/cbor"
 )
 
 var HeaderMagicBytes = []byte{0x84, 0x48, 0xf0, 0x9f, 0x8c, 0x90, 0xf0, 0x9f, 0x93, 0xa6}
 
+type Request struct {
+	*url.URL
+	http.Header
+}
+
+type Response struct {
+	Status string
```

Comment:
As HTTP status is represented as an integer in `net/http` package, let's use an integer here.